dist/api/bundler.js

```diff
@@ -0,0 +1,113 @@
+'use strict';
+
+var api = require('jspm/api');
+var whacko = require('whacko');
+var glob = require('glob');
+var fs = require('fs');
+var url = require('url');
+var path = require('path');
+
+var pluginName = 'view';
+var loader = api.Builder().loader;
```

Comment:
This global declaration needs to be moved into the functions needing this, as this code expects the destination to have a package.json file, which is not happening on the first run